cmd/entrypoint/main.go

```diff
@@ -75,14 +77,16 @@ type RealWaiter struct{}
 
 var _ entrypoint.Waiter = (*RealWaiter)(nil)
 
-func (*RealWaiter) Wait(file string) error {
+func (*RealWaiter) Wait(file string, expectContent bool) error {
 	if file == "" {
 		return nil
 	}
 	for ; ; time.Sleep(time.Second) {
 		// Watch for the post file
```

Comment:
can we expand this comment here to elaborate on what we expect to populate the file, and why, and with what?

Reply:
I've removed the two inline comments and added a godoc to this func that explains the behaviour and edge cases.